doc/doxygen-layout-user.xml

```diff
@@ -77,10 +77,10 @@
       <typedefs title=""/>
       <functions title=""/>
       <properties title="OpenSim Property, Connector, Output, Input Documentation"/>
-      <!-- <enums title=""/>
+      <enums title=""/>
       
       
-      <related title=""/>
+      <!-- <related title=""/>
```

Comment:
I'm pretty sure this means that the enums appear at the very very bottom of the pages. Is this desirable? Should they also be under the declaration section (`<memberdecl>`)? Have you checked to see how this affects the documentation of any other enums? Are there cryptic enums that should be hidden from users that are now shown?